test/util/k8sreporter/descriptions.go

```diff
@@ -0,0 +1,42 @@
+package k8sreporter
+
+import (
+	"context"
+	"fmt"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
```

Comment:
nit: all these imports need to be in different groups ? 

specifically controller-runtime and core/v1

Reply:
I can definitely group them. fixing